sass/digital-exchange/common/SidebarFilter.scss

```diff
@@ -0,0 +1,16 @@
+.SidebarFilter {
+  text-align: left;
+}
+
+
+.SidebarFilter__title {
+  color: #4D5258;
+  font-family: Open Sans;
+  font-style: normal;
+  font-weight: bold;
+  line-height: normal;
+  font-size: 12px;
+  background-color: #E7E7E7;
```

Comment:
We should use SASS variables for colors: it improves readability, consistency and it's useful if we want to support theming in DE (@nicpuddu is it planned AFAYK?)